### ISSUE TITLE
How to delete consumer group (using new consumer API) in kafka manger?

### ISSUE THREAD

**Issue by KaimingWan** [yahoo/kafka-manager#341]
As we know, kafka 0.10.x store consumer's offset in a topic. We can't delete a group id in kafka manager by remove the node in zookeeper. However, I have a deprecated consumer group but the topic it consumerd from is still useful. We can see the lag of the consumer group  is keep growing. Are there some method to work this out?

**Comment by wufsmail** [comment]
you can disabled the kafka manager, and enable it.

**Comment by songxin1990** [comment]
good question。

**Comment by ksilin** [comment]
an option to delete or reset a consumer group would be great

**Comment by ThomasLau** [comment]
Did anybody solved this problem?i met the same question that i can see the deleted consumer group in Kafka 0.9

**Comment by asyd** [comment]
Any news by chance?

**Comment by geez** [comment]
You guys can do that with burrow, look at the API: https://github.com/linkedin/Burrow/wiki/http-request-remove-consumer-group

Burrow: https://github.com/linkedin/Burrow
API: https://github.com/linkedin/Burrow/wiki/HTTP-Endpoint

**Comment by imjustfly** [comment]
@geez 

No. This api just make Burrow not check the lag of that group. It doesn't remove that group from kafka broker.

**Comment by deric** [comment]
There's no need to delete group metadata for the new consumer as the group is deleted when the last committed offset for that group expires.

**Comment by Ian-lin-max** [comment]
@deric The consumer group can be automatically deleted？

**Comment by dungmv** [comment]
any update
